taskt/Core/Automation/Commands/IE Browser Commands/IECloseBrowserCommand.cs

```diff
@@ -53,7 +59,7 @@ public override List<Control> Render(frmCommandEditor editor)
 
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [Instance Name: '" + v_InstanceName + "']";
+            return base.GetDisplayValue() + $" [Close IE Browser Instance '{v_InstanceName}']";
```

Comment:
Could you please follow the same format for displaying instances that I used in Excel and Word? So for example this would just be [Instance Name '{Instance Name}']

Reply:
done